### ISSUE TITLE
Could not pass event AsyncPlayerChatEvent to CensorReloaded v3.9.7

### ISSUE THREAD

**Issue by RalphORama** [EncryptSL/CENSOR-RELOADED-ISSUES#2]
server version: `git-Paper-209 (MC: 1.14.4) (Implementing API version 1.14.4-R0.1-SNAPSHOT)`

I get the following error in console occasionally.  Not sure what the issue is but I figured I'd report it.

```
[21:10:38 ERROR]: Could not pass event AsyncPlayerChatEvent to CensorReloaded v3.9.7
java.lang.NullPointerException: null
at org.encryptsl.censor.filters.BadWords.badWordsFilter(BadWords.java:31) ~[?:?]
at org.encryptsl.censor.listener.AsyncChatEventClass.onAsyncChat(AsyncChatEventClass.java:18) ~[?:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor22.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.14.4.jar:git-Paper-209]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.14.4.jar:git-Paper-209]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-209]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-209]
at net.minecraft.server.v1_14_R1.PlayerConnection.chat(PlayerConnection.java:1713) ~[patched_1.14.4.jar:git-Paper-209]
at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1628) ~[patched_1.14.4.jar:git-Paper-209]
at net.minecraft.server.v1_14_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:41) ~[patched_1.14.4.jar:git-Paper-209]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_212]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_212]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_212]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_212]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

**Comment by EncryptSL** [comment]
@RalphORama hello sir, you have in your plugins folder generated folder CensorReloaded and
with config.yml and blockedwords.txt.

You have another chatdetection plugins with using AsyncChatEvent or plugin using same detect functions ? 
You can send your plugin screenshot or list with your plugins thanks. 

On version  ` This server is running Paper version git-Paper-210 (MC: 1.14.4) (Implementing API version 1.14.4-R0.1-SNAPSHOT)`
![image](https://user-images.githubusercontent.com/9441083/66612137-1977ae00-ebc1-11e9-82f2-78957eefd491.png)

Plugin works normal, i think something wrong is on your server side or in blockedwords.text.

**Comment by RalphORama** [comment]
Hello,

Here's my plugin list:

-  CensorReloaded
-  ColoredPlayerNames
-  CoreProtect
-  DeathMessagesPrime
-  dynmap*
-  Dynmap-Essentials*
-  Essentials
-  EssentialsChat
-  EssentialsGeoIP
-  EssentialsSpawn
-  GameModeInventories
-  HiddenLorax
-  LuckPerms
-  Matrix
-  mcMMO
-  PlaceholderAPI
-  Plan
-  ProtocolLib
-  RandomTeleport
-  ServerListPlus*
-  TimeControl
-  Vault
-  VeinMiner
-  ViaVersion*
-  WorldBorder

And my blockedwords.txt:

```regex
\b(a+(\W|\d|_)*b+(\W|\d|_)*b+(\W|\d|_)*o)
\b(a+(\W|\d|_)*b+(\W|\d|_)*o)
\b(a+(\W|\d|_)*u+(\W|\d|_)*t+(\W|\d|_)*i+(\W|\d|_)*s+(\W|\d|_)*m)
\b(a+(\W|\d|_)*u+(\W|\d|_)*t+(\W|\d|_)*i+(\W|\d|_)*s+(\W|\d|_)*t+(\W|\d|_)*i+(\W|\d|_)*c)
\b(b+(\W|\d|_)*e+(\W|\d|_)*a+(\W|\d|_)*n+(\W|\d|_)*e+(\W|\d|_)*r)
\b(b+(\W|\d|_)*i+(\W|\d|_)*t+(\W|\d|_)*c+(\W|\d|_)*h)
\b(c+(\W|\d|_)*h+(\W|\d|_)*i+(\W|\d|_)*n+(\W|\d|_)*k)
\b(c+(\W|\d|_)*o+(\W|\d|_)*o+(\W|\d|_)*n)
\b(c+(\W|\d|_)*u+(\W|\d|_)*n+(\W|\d|_)*t)
\b(d+(\W|\d|_)*y+(\W|\d|_)*k+(\W|\d|_)*e)
\b(f+(\W|\d|_)*a+(\W|\d|_)*g)
\b(f+(\W|\d|_)*a+(\W|\d|_)*g+(\W|\d|_)*g+(\W|\d|_)*o+(\W|\d|_)*t)
\b(f+(\W|\d|_)*g+(\W|\d|_)*t)
\b(j+(\W|\d|_)*a+(\W|\d|_)*p)
\b(k+(\W|\d|_)*i+(\W|\d|_)*k+(\W|\d|_)*e)
\b(l+(\W|\d|_)*o+(\W|\d|_)*l+(\W|\d|_)*i)
\b(n+(\W|\d|_)*i+(\W|\d|_)*g)
\b(n+(\W|\d|_)*i+(\W|\d|_)*g+(\W|\d|_)*g+(\W|\d|_)*a)
\b(n+(\W|\d|_)*i+(\W|\d|_)*g+(\W|\d|_)*g+(\W|\d|_)*e+(\W|\d|_)*r)
\b(n+(\W|\d|_)*i+(\W|\d|_)*p)
\b(t+(\W|\d|_)*a+(\W|\d|_)*r+(\W|\d|_)*d)
\b(r+(\W|\d|_)*e+(\W|\d|_)*t+(\W|\d|_)*a+(\W|\d|_)*r+(\W|\d|_)*d)
\b(r+(\W|\d|_)*e+(\W|\d|_)*t+(\W|\d|_)*a+(\W|\d|_)*r+(\W|\d|_)*d+(\W|\d|_)*e+(\W|\d|_)*d)
\b(s+(\W|\d|_)*l+(\W|\d|_)*a+(\W|\d|_)*n+(\W|\d|_)*t)
\b(s+(\W|\d|_)*l+(\W|\d|_)*u+(\W|\d|_)*t)
\b(s+(\W|\d|_)*p+(\W|\d|_)*i+(\W|\d|_)*c)
\b(t+(\W|\d|_)*r+(\W|\d|_)*a+(\W|\d|_)*n+(\W|\d|_)*n+(\W|\d|_)*y)
\b(t+(\W|\d|_)*r+(\W|\d|_)*a+(\W|\d|_)*p)
\b(w+(\W|\d|_)*e+(\W|\d|_)*t+(\W|\d|_)*b+(\W|\d|_)*a+(\W|\d|_)*c+(\W|\d|_)*k)
\b(w+(\W|\d|_)*h+(\W|\d|_)*o+(\W|\d|_)*r+(\W|\d|_)*e)
```

**Comment by EncryptSL** [comment]
Hello sir we are looking on this error but not found what is bad.  
I would have helped you with this problem but nowhere have we noticed a bug and we have tested everything on the latest build on server without plugins. And with yours list blockedwords.yml

Your plugins looks good but I think problem is in ViaVersion or Death Messages Prime.

I not locking this topic but you must test whats in wrong on my side not problem is with another plugins.

Before can you create backup yours blockedwords.yml, and delete complete CensorReloaded plugin and folder. Stop server and you can again upload CensorReloaded plugin and when plugin generated folder with this plugin you can upload now blockedwords.yml.

**Comment by EncryptSL** [comment]
Possible fixed in 4.0.0 IDK, you must test this.
